internal/svg/svg.go

```diff
@@ -22,22 +22,28 @@ import (
 
 // Colorize creates a new SVG from a given one by replacing all fill colors by the given color.
 func Colorize(src []byte, clr color.Color) []byte {
+	content, err := ColorizeError(src, clr)
+	if err != nil {
+		fyne.LogError("", err)
+	}
+	return content
+}
+
+// ColorizeError is the same as Colorize, except returning instead of logging any error.
+func ColorizeError(src []byte, clr color.Color) ([]byte, error) {
```

Comment:
Sorry for the pickiness but I'm not sure about the function name here. ColorizeError sounds as if it is changing the colour of an error and not an svg?

Reply:
Do you have a better name in mind? `ColorizeReturnError`?

Reply:
Not really, no. I think having in the name that something returns an error will be strange. Also the idea of having a version that returns errors and one that doesn't is strange in most cases. I think there are two options:

1. Depending on how many call sites there are, it might make sense to only use the version that returns errors and either look at it or ignore it completely.
2. The standard library seems to follow the convention that a function always returns an error and in the case where there is a version of it that doesn't, it is called MustXYZ (although the latter usually panics on errors).

It is a bit of a strange pattern in general come to think of it. Any ideas?

Reply:
I would agree with 1 - there is little point in exposing the non-error as a public helper API if errors can be generated.